Controller/MediaAdminController.php

```diff
@@ -77,10 +77,11 @@ public function listAction(Request $request = null)
         $datagrid->setValue('context', null, $context);
 
         // retrieve the main category for the tree view
-        $category = $this->container->get('sonata.classification.manager.category')->getRootCategory($context);
+        $rootCategory = $this->container->get('sonata.classification.manager.category')->getRootCategory($context); 
```

Comment:
I might look a bit retarded now but i still don't see it :P 
<img width="1081" alt="screenshot 2017-01-16 15 12 57" src="https://cloud.githubusercontent.com/assets/1973973/21986105/5343f5ba-dbfe-11e6-89dd-4bb47e6fcc84.png">